docs/serving/queue-extensions.md

```diff
@@ -0,0 +1,31 @@
+# Queue Proxy Runtime Extensions
```

Comment:
this title is not really descriptive; it doesn't tell a user why they would want to read this section or what information to expect